README.md

```diff
@@ -11,9 +11,98 @@ and install the configured UI extension. All configuration is provided
+|--------------------------|-----------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| EXTENSION_NAME           | Yes       | ""      | Extension Name                                                                                                                                                                                                                              |
+| EXTENSION_ENABLED        | No        | true    | If set to false will skip the installation. Noop                                                                                                                                                                                            |
+| EXTENSION_URL            | Yes       | ""      | Must be set to a valid URL where the UI extension can be downloaded from. <br>Argo CD API server needs to have network access to this URL.                                                                                                  |
+| EXTENSION_VERSION        | Yes       | ""      | The version of the extension to be installed.                                                                                                                                                                                               |
+| EXTENSION_CHECKSUM_URL   | No        | ""      | Can be set to the file containing the checksum to validate the downloaded<br>extension. Will skip the checksum validation if not provided.<br>Argo CD API server needs to have network access to this URL.                                  |
+| MAX_DOWNLOAD_SEC         | No        | 30      | Total time in seconds allowed to download the extension.                                                                                                                                                                                    |
+| EXTENSION_VARS           | No        | ""      | Specifies the variables to be exported in $EXTENSION_VARS_FILE_NAME in json format within the extension folder. These variables serve as external configurations for the extension. <br/>The format should be `{key1=value1, key2=value2}`. |
+| EXTENSION_VARS_FILE_NAME | No        | ''      | Specifies the file name where the variables will be exported in json format. You can provide your own file name, e.g., 'config'.                                                                                                            |
+| EXTENSION_VARS_FILE_PATH | No        | ""      | Path to export variables, e.g., '/tmp/extensions/resources/extension-{EXTENSION_NAME}.js'. Ensure `EXTENSION_NAME` matches the extension name.                                                                                              |
```

Comment:
I think that we shouldn't expose these 2 variables. Instead, we should define our own convention about how files will be named and where they will be placed.

Reply:
done